### PR TITLE
User Application Refine

### PR DESCRIPTION
**Pull request by mindnervestech**

### CODE REVIEW

File: app/app.controller.js [hunk 1]
```diff
@@ -16,39 +16,58 @@ function ApplicationController($scope, UserApplicationService, $state, $rootScop
       $scope.$apply()
    	}
 
-	  vm.currentUser = null;
-    vm.setCurrentUser = function(user) {
-      vm.currentUser = user;
-    };
+	      
+    /*vm.getUsersavedStageData = function () {
```
Comment (oolawo09) [inline app/app.controller.js]: could you remove this commented code? 

If it appears on an earlier commit then you could always retrieve it from there when you need it

File: app/app.controller.js [hunk 2]
```diff
@@ -16,39 +16,58 @@ function ApplicationController($scope, UserApplicationService, $state, $rootScop
...
+        console.log("selected stage name", stage.stageName);
+        vm.stageName = stage.stageName;
+    });
+
+    $scope.$on("submittedStageProgress", function(event, progress){
+        console.log("selected stage name", progress.progress);
+        vm.stageProgress = progress.progress;
+    });
+
+  	vm.getApplicationData  = function (applicatioName) {
```
Comment (oolawo09) [inline app/app.controller.js]: also auto indent your code so that it's easier to read

File: app/core/authenticate/authenticate.service.js [hunk 3]
```diff
@@ -73,11 +73,52 @@ angular.module("core.authenticate")
             return deferred.promise;
         }
 
+        function setProfile(email, data) {
```
Comment (oolawo09) [inline app/core/authenticate/authenticate.service.js]: could you move this to the profile.service.js file ? That way all the code to manipulate profiles is in one place

File: app/app.controller.js (hunk 1 above)
Comment (mindnervestech) [reply app/app.controller.js]: completed

File: app/core/authenticate/authenticate.service.js (hunk 3 above)
Comment (mindnervestech) [reply app/core/authenticate/authenticate.service.js]: completed

File: app/app.controller.js (hunk 2 above)
Comment (mindnervestech) [reply app/app.controller.js]: completed

**Pull request merged**